examples/fluent-bit/oomkill-prevention/README.md

```diff
@@ -210,6 +211,52 @@ Also, there is an error message when new chunks can not be ingested at all due t
 [error] [input chunk] chunk 1-1678829080.465983679.flb would exceed total limit size in plugin cloudwatch_logs.0
 ```
 
+### Full FireLens Configuration Examples
+
+* [Full Filesystem Storage Example](firelens-full-filesystem-example/)
+* [Full Memory Buffer Example](firelens-full-memory-example/)
+
+These examples show how to control the memory usage for all inputs in FireLens, including the FireLens stdout/stderr unix socket forward input. These examples use the technique in [How to set Fluentd and Fluent Bit input parameters in FireLens](https://aws.amazon.com/blogs/containers/how-to-set-fluentd-and-fluent-bit-input-parameters-in-firelens/). The entrypoint for the Fluent Bit container is customized to use an alternate configuration file path and ignore the FireLens generated input. 
+
+Please note, this technique ignores & overrides the FireLens generated configuration and thus has the following downsides:
+* All containers that use Fluent Bit for stdout and stderr logging should specify the `awsfirelens` log driver without any options. 
+* All configuration must be present in your custom config file; the `logConfiguration` options map should be empty as its content goes to the FireLens generated configuration.
+* The `enable-ecs-log-metadata` firelens setting (default true) should be set to `false` because the metadata goes to the FireLens generated configuration file. Your logs will not have metadata attached: [What will the logs collected by FireLens look like?](https://github.com/aws/aws-for-fluent-bit/blob/mainline/troubleshooting/debugging.md#what-will-the-logs-collected-by-fluent-bit-look-like). To add metadata to logs, consider using the [init tag with ECS metadata support in env vars](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/init-metadata).
+
+Please review the full configuration examples carefully. The following is of note:
+* The FireLens configuration object does not specify a config file to import:
+```
+"firelensConfiguration": {
+	"type": "fluentbit",
+	"options":{
+		"enable-ecs-log-metadata":"false"
+	}
+}
+```
+* The app container specifies `awsfirelens` with no options:
+```
+"logConfiguration": {
+	"logDriver":"awsfirelens"
+}
+```
+* The Dockerfile for Fluent Bit adds our custom config file and overrides the entrypoint to use it:
+```
+FROM public.ecr.aws/aws-observability/aws-for-fluent-bit:stable
+ADD extra.conf /fluent-bit/alt/fluent-bit.conf
+CMD echo -n "AWS for Fluent Bit Container Image Version " && \
+    cat /AWS_FOR_FLUENT_BIT_VERSION && echo "" && \
+    exec /fluent-bit/bin/fluent-bit -e /fluent-bit/firehose.so -e /fluent-bit/cloudwatch.so -e /fluent-bit/kinesis.so -c /fluent-bit/alt/fluent-bit.conf
```

Comment:
is this the necessary step to change the entry point? As the custom config is in a custom location?

Reply:
Yes this is required.